### DISCUSSION TITLE
Export to Image (PNG or JPG) via dom-to-image-more

### DISCUSSION THREAD

**Discussion by shadowcpy** [directus/directus#7623] [Feature Requests]
How about instead of trying to do export to pdf with templates we add the option in the export dialog to export an image of the current collections dataset (with orientation selection) and then one can just embed it into a pdf if it is needed. 
I would propose using a package like https://www.npmjs.com/package/dom-to-image-more to capture the relevant information (just the table, without dialogs etc.. and then downloading the image.
What do you think?

**Comment by benhaynes** [comment]
Couldn't we also use the `@media` print CSS to have proper print support? From the system print dialog you can save as PDF, and we already have CSS overrides, so users could make changes (instead of complex templates).

**Comment by rijkvanzanten** [comment]
The downside with that is that you can't copy (textual) data from an image export, so it would severely limit the usability of the output pdf etc.

That being said, it would be nice option to have for the Map / Calendar layout and the upcoming Insights module

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
